ts/Core/Tooltip.ts

```diff
@@ -69,6 +69,7 @@ declare module './Series/PointLike' {
 declare module './Series/SeriesLike' {
     interface SeriesLike {
         noSharedTooltip?: boolean;
+        noSplitTooltip?: boolean;
```

Comment:
Probably better to check `noSharedTooltip` for split tooltip too. A split tooltip is also a form of shared tooltip, in the sense that multiple series are shown together. Also, `noSharedTooltip` is not limited to pie series.

Reply:
I decided to use `noSplitTooltip` because series like for example `scatter` might have splitted tooltip, when shared doesn't work for them.

Reply:
Unfortunately, it is the case for the flag series.
I'm afraid we have to have separate properties for that.

Reply:
As I recall it was never the intention that flags should have a split tooltip. I think it just happened and has been patched after. I'm pushing a commit with some simplifications.